docs/sources/http_api/folder.md

```diff
@@ -28,7 +28,7 @@ that you cannot use this API for retrieving information about the General folder
 
 `GET /api/folders`
 
-Returns all folders that the authenticated user has permission to view.
+Returns all folders that the authenticated user has permission to view. You can control the maximum number of folders returned through the `limit` query parameter, the default is 1000.
```

Comment:
```suggestion
Returns all folders that the authenticated user has permission to view. You can control the maximum number of folders returned through the `limit` query parameter, the default maximum limit is 1000.
```

Reply:
The issue that this fixes said that they expected the default to be all folders. I think we should clarify that if they want to remove the limits, they cannot, but they can set the limit to higher than their current number of folders.

Reply:
@oddlittlebird I don't think your suggested edit makes sense to be honest. The original wording describes it correctly - the default for the "limit" parameter is 1000 and it doesn't control the "maximum limit", but rather the limit for how many folders to return (i.e. <= limit).

@sjoeboo Since you authored the issue solved by this PR, is there anything you would want different about the wording?

Reply:
@marefr @bergquist @papagian Do you prefer my wording or the suggested edit here?

Reply:
I would find `maximum limit` confusing and take it to mean "the most results you can expect, regardless of the passed query limit".

The original change ` You can control the maximum number of folders returned through the `limit` query parameter, the default is 1000.` is totally fine with me.

Reply:
I think we're on the same page then @sjoeboo :)